### PR TITLE
cmd/osbuild-worker: Configure s3 bucket on the worker itself

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

File: templates/packer/ansible/roles/common/files/worker-initialization-scripts/worker_service.sh [hunk 1]
```diff
@@ -15,6 +15,7 @@ credentials = "/etc/osbuild-worker/gcp_credentials.json"
 credentials = "/etc/osbuild-worker/azure_credentials.toml"
 [aws]
 credentials = "/etc/osbuild-worker/aws_credentials.toml"
+bucket = "${WORKER_CONFIG_AWS_BUCKET:-}"
```
Comment (ondrejbudai) [inline templates/packer/ansible/roles/common/files/worker-initialization-scripts/worker_service.sh:18]: We will have to set this in cloud-init, right?

**Review by ondrejbudai — APPROVED** [review]

File: templates/packer/ansible/roles/common/files/worker-initialization-scripts/worker_service.sh (hunk 1 above)
Comment (Gundersanne) [reply templates/packer/ansible/roles/common/files/worker-initialization-scripts/worker_service.sh:18]: Yup, we can do that in app-intf/image-builder-tf and have each worker asg have a different bucket.

**Comment by Gundersanne** [comment]
packit is broken? :cry:

**Comment by Gundersanne** [comment]
Also parameterized the aws credentials config, as we need to receive the s3 access policy from the iam role.

**Review by ondrejbudai — APPROVED** [review]

**Pull request merged**